src/pages/Home/Home.page.jsx

```diff
@@ -1,36 +1,39 @@
 import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+// import { Link, useHistory } from 'react-router-dom';
 
 import { useAuth } from '../../providers/Auth';
 import './Home.styles.css';
+import VideoCardGrid from '../../components/VideoCardGrid';
 
 function HomePage() {
-  const history = useHistory();
+  // const history = useHistory();
   const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
+  const { authenticated } = useAuth();
 
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
+  // function deAuthenticate(event) {
+  //   event.preventDefault();
+  //   logout();
+  //   history.push('/');
+  // }
 
   return (
     <section className="homepage" ref={sectionRef}>
-      <h1>Hello stranger!</h1>
       {authenticated ? (
         <>
-          <h2>Good to have you back</h2>
+          {/* <h2>Good to have you back</h2>
```

Comment:
I know it is a work in progress, but just to point out, try to remove the comment code